src/autoscaler/integration/components_test.go

```diff
@@ -100,6 +116,18 @@ func (components *Components) GolangAPIServer(confPath string, argv ...string) *
 		),
 	})
 }
+
+func (components *Components) GolangAPICFServer(argv ...string) *ginkgomon_v2.Runner {
+	return ginkgomon_v2.New(ginkgomon_v2.Config{
+		Name:              GolangAPIServer,
+		AnsiColorCode:     "33m",
+		StartCheck:        "api.started",
```

Comment:
how about "apiCFServer.started" to differentiate between bosh and cf services?

Reply:
Currently, we don’t log whether the configuration is loaded from environment variables (used in Cloud Foundry) or from a YAML file (used in VMs). We simply proceed as long as the configuration passes validation, regardless of its source, and we error out if it doesn’t.